.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+# Simple workflow for building documentation
+name: Build documentation
+
+# On a pull request this will build the html documentation
+# and create an artifact. It does this by using the standard
+# build target doc_html
+
+on:
+ pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+ workflow_dispatch:
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+
+      - name: Install apt package
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y doxygen spirv-tools virtualenv
+
+      - name: Install prerequisites
+        run: python3 -m pip install -r doc/requirements.txt
```

Comment:
Do we need `python3` here or will `setup-python` create `python = python3` for us? Just trying to reduce the number of places we're encoding the version.